exchange/bidder_test.go

```diff
@@ -1973,3 +2017,10 @@ func (bidder *notifyingBidder) MakeBids(internalRequest *openrtb2.BidRequest, ex
 func (bidder *notifyingBidder) MakeTimeoutNotification(req *adapters.RequestData) (*adapters.RequestData, []error) {
 	return &bidder.notifyRequest, nil
 }
+
+type MockInProcessSigner struct {
```

Comment:
Can we just call this MockSigner? Does it being "in process" mean anything to the test?

Reply:
Right, replaced to MockSigner and removed MockInProcessSigner. Good catch!